pkg/client/unversioned/extensions.go

```diff
@@ -130,24 +130,24 @@ func NewExtensionsOrDie(c *Config) *ExtensionsClient {
 
 func setExtensionsDefaults(config *Config) error {
 	// if experimental group is not registered, return an error
-	g, err := latest.Group("extensions")
-	if err != nil {
-		return err
+	if !extlatest.IsEnabled() {
```

Comment:
> Which groups are enabled is a runtime option, not a compile-time option, so it's not appropriate to use the import graph for determining this.

I agree, but this type is explicitly made to operate against exactly one API group.  The lookup simply masked that fact.  I didn't introduce the dependency, I only made the dependency obvious from the code.